blimp-live-widgets/src/components/Weather/index.js

```diff
@@ -58,17 +63,39 @@ export default class Weather extends Component {
 
   render() {
     const {error, isLoaded, weather} = this.state;
+    var img_src = sunny
+    var temp_unit = "°C"
+    if (weather.units == "metric") {
+      temp_unit = "°C"
+    } else {
+      temp_unit = "°F"
+    }
+    if (weather.main == "Clouds") {
+      img_src = cloud
+    } else if (weather.main == "Clear") {
+      img_src = sunny
+    }
+    else if (weather.main == "Rain") {
+      img_src = rain
+    }
+    else if (weather.main == "Snow") {
+      img_src = snow
+    }
+    else if (weather.main == "Thunderstorm") {
```

Comment:
Is it possible to have other options? Like did we cover all cases or is there a chance that we won't have an image sometime? Overall, I guess my question is do we need an else statement as like a catch all?

Reply:
The variable is set to sunny to begin with which can be treated as the catch all.